kubernetes_tests/test_cleanup_pods.py

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import unittest
+from unittest import mock
+
+from airflow.cli import cli_parser
+from airflow.cli.commands import kubernetes_command
+from airflow.kubernetes.kube_client import get_kube_client
+
+
+class TestCleanupPods(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.parser = cli_parser.get_parser()
+
+    @mock.patch("airflow.cli.commands.kubernetes_command.get_kube_client")
+    def test_cleanup_pods_working(self, mock_client):
+        mock_client.return_value = get_kube_client(in_cluster=False)
+        kubernetes_command.cleanup_pods(
+            self.parser.parse_args(['kubernetes', 'cleanup-pods', '--namespace', 'test-namespace'])
+        )
```

Comment:
We already have tests at https://github.com/apache/airflow/blob/9d6c7487dd03a42d1b6a1f19fae0c9a746139dbe/tests/cli/commands/test_kubernetes_command.py#L57-L174

Can you add a test over there please

Reply:
I think this test different with unit tests, this case maybe test with Kubernetes

Reply:
@Junnplus You are also mocking `get_kube_client` in the test, so it is not run on the K8s cluster

Reply:
@kaxil I remove this k8s test, but I dont know how to do this unit test

Reply:
We don't support python client 12.0.0. https://github.com/apache/airflow/blob/065c02b3614d7839c0bf62c5aa467cac59293aea/constraints-3.8.txt#L223 See also: https://github.com/kubernetes-client/python/issues/1304

Reply:
https://github.com/apache/airflow/pull/13563#discussion_r555131890

https://github.com/kubernetes-client/python/blob/v11.0.0/kubernetes/client/models/v1_list_meta.py#L56-L57

https://github.com/kubernetes-client/python/blob/v12.0.0/kubernetes/client/models/v1_list_meta.py#L61-L62

The code looks same for v11.0.0 and v12.0.0 anyways.. so are you sure an empty string works @Junnplus  ?

Reply:
@mik-laj I know we use kubernetes python client is 11.0
> _continue is None will raise above error in kubernetes python 11.0 .

for 11.0
https://raw.githubusercontent.com/kubernetes-client/python/v11.0.0/kubernetes/client/api/core_v1_api.py
```
    def list_pod_for_all_namespaces_with_http_info(self, **kwargs):  # noqa: E501
...
        if '_continue' in local_var_params:
            query_params.append(('continue', local_var_params['_continue']))  # noqa: E501
```
for 12.0
https://raw.githubusercontent.com/kubernetes-client/python/v12.0.0/kubernetes/client/api/core_v1_api.py
```
    def list_pod_for_all_namespaces_with_http_info(self, **kwargs):  # noqa: E501
```
        if '_continue' in local_var_params and local_var_params['_continue'] is not None:  # noqa: E501
            query_params.append(('continue', local_var_params['_continue']))  # noqa: E501
```